blog/2020-11-06-project-feast-closing-notes.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Fail⬅️⬅️⬅️⬅️Learning faster"
+author: franck
+author_url: https://github.com/d4nte
+author_image_url: https://avatars1.githubusercontent.com/u/300805
+tags: [bitcoin,dai,atomic swap,ambrosia]
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Ambrosia
+
+We recently released [ambrosia](./2020-10-12-ambrosia-alpha-release.md), a GUI allowing BTC to DAI peer-to-peer and non-custodial trading powered by COMIT.
+This release marked our change of focus from providing tools to developer to directly targeting end users. 
+
+We spent the last weeks demonstrating this MVP to several users, we had interesting feedback and conversation with various user profiles, from trader to hodler.
+
+<!--truncate-->
+
+At this point of time, we felt that BTC&lt;&gt;DAI dex trading may not be what users are currently after and hence decided to drop the project.
+
+While the development efforts which lead to this MVP did not lead to a successful product, time was not wasted.
+We made major learnings that will allow us to move faster, for example, our blockchain monitoring solution was not up to scratch for a mainnet environment.
+You can find more details of our learnings in the [closing notes](https://lists.comit.network/pipermail/comit-dev/2020-October/000017.html).
+
+# wBTC&lt;&gt;BTC
```

Comment:
If you don't want to have these weird characters, you could write this: `BTC/DAI` 
```suggestion
# wBTC/BTC
```

Reply:
Actually this looks quite good as it is.

![image](https://user-images.githubusercontent.com/224613/98485725-a41b5880-226c-11eb-8820-cbf04e48c6a3.png)